cmd/app.go

```diff
@@ -72,10 +80,21 @@ func parseFlags(osArgs []string) (*appConfig, error) {
 	flagSet.Var(&config.logLevel, "log-level", "Set the log level according to zapcore.Level")
 	flagSet.BoolVar(&config.enableMetrics, "metrics", false, "Enable Prometheus HTTP /metrics server")
 	flagSet.BoolVar(&config.enableHealthz, "healthz", false, "Enable HTTP health endpoints server")
+	flagSet.BoolVar(&config.enableAuditMetrics, "audit-metrics", false, "Enable Prometheus audit metrics")
 	flagSet.DurationVar(&config.httpServerReadTimeout, "http-server-read-timeout",
 		DefaultHTTPServerReadTimeout, "HTTP server read timeout")
 	flagSet.DurationVar(&config.httpServerReadHeaderTimeout, "http-server-read-header-timeout",
 		DefaultHTTPServerReadHeaderTimeout, "HTTP server read header timeout")
+	flagSet.IntVar(
+		&config.auditMetricsSecondsInterval,
+		"audit-seconds-interval",
+		DefaultAuditCheckInterval,
+		"Interval in seconds to collect audit metrics")
+	flagSet.IntVar(
```

Comment:
I think a `time.Duration` would be more flexible here (via `FlagSet.DurationVar`).

Reply:
will do

Reply:
need a int here cause we compare seconds in the func.